com.unity.ml-agents/Tests/Editor/ParameterLoaderTest.cs

```diff
@@ -157,7 +157,7 @@ public void TestGetOutputTensors1()
         {
             var model = ModelLoader.Load(continuous2vis8vec2actionModel);
```

Comment:
I think you made a lot of improvements to the model loading logic. I think you should add more tests to check your logic (loading an old model and a new model and see if they behave properly)

Reply:
More tests with new models added